client/my-sites/guided-transfer/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import GuidedTransfer from './guided-transfer';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+function mapStateToProps( state ) {
+	return {
+		showExporter: () => {
+			const siteId = getSelectedSiteId( state );
+			const siteSlug = getSiteSlug( state, siteId );
+
+			page( `/settings/export/${siteSlug}` );
+		}
```

Comment:
Likewise, here we could pass the `siteSlug` as the prop and implement the `showExporter` function inside the component itself